std_msgs/Header header
float32 rotor1
float32 rotor2
float32 rotor3
float32 rotor4
